x/staking/src/params/mod.rs

```diff
@@ -34,6 +35,32 @@ pub struct StakingParams {
     pub bond_denom: Denom,
 }
 
+/// [`RawParams`] exists to allow us to validate params when deserializing them
+#[derive(Deserialize)]
+struct RawStakingParams {
+    unbonding_time: Duration,
+    max_validators: u32,
+    max_entries: u32,
+    historical_entries: u32,
+    bond_denom: Denom,
+}
+
+impl TryFrom<RawStakingParams> for StakingParams {
+    type Error = anyhow::Error;
+
+    fn try_from(params: RawStakingParams) -> Result<Self, Self::Error> {
+        StakingParams::new(
+            i128::from(params.unbonding_time.duration_nanoseconds())
```

Comment:
Can we make the `StakingParams::new` method's `unbonding_time` argument type be `Duration`? We might need to derive `Ord` on `Duration`?